src-ts/lib/work-provider/work-functions/work-factory/work.factory.ts

```diff
@@ -289,3 +289,77 @@ function getTypeCategory(type: WorkType): WorkTypeCategory {
             return WorkTypeCategory.unknown
     }
 }
+
+export function mapFormData(type: string, formData: any): any {
```

Comment:
for future reference, i'd like to keep the functions ordered with exported functions first followed by private functions, and each group sorted alphabetically. it just makes it easier to find things and to quickly understand scope.